flink-rpc/flink-rpc-akka/src/main/java/org/apache/flink/runtime/rpc/akka/AkkaUtils.java

```diff
@@ -433,23 +434,23 @@ public static ActorSystem createDefaultActorSystem() {
     }
 
     /**
-     * Creates the default akka configuration which listens on a random port on the local machine.
+     * Creates the default pekko configuration which listens on a random port on the local machine.
      * All configuration values are set to default values.
      *
-     * @return Flink's Akka default config
+     * @return Flink's Pekko default config
      */
     private static Config getDefaultAkkaConfig() {
         return getAkkaConfig(new Configuration(), new HostAndPort("", 0));
     }
 
     /**
-     * Creates an akka config with the provided configuration values. If the listening address is
+     * Creates a pekko config with the provided configuration values. If the listening address is
      * specified, then the actor system will listen on the respective address.
      *
      * @param configuration instance containing the user provided configuration values
      * @param externalAddress optional tuple of bindAddress and port to be reachable at. If null is
-     *     given, then an Akka config for local actor system will be returned
-     * @return Akka config
+     *     given, then an Pekko config for local actor system will be returned
```

Comment:
```suggestion
     *     given, then a Pekko config for local actor system will be returned
```
nit: since you fixed it properly in the JavaDoc above :man_shrugging: